Gems/AWSCore/Code/Tools/ResourceMappingTool/README.md

```diff
@@ -49,10 +54,9 @@ Follow cmake instructions to configure your project, for example:
       $ python/python.sh Gems/AWSCore/Code/Tools/ResourceMappingTool/resource_mapping_tool.py --binaries_path <PATH_TO_BUILD_FOLDER>/bin/debug/AWSCoreEditorQtBin
       ```
       
-* Note - Editor is integrated with the same engine python environment to launch Resource Mapping Tool. If it is failed to launch the tool
-in Editor, please follow above steps to make sure expected scripts/binaries are present.
+* Note - the Editor is integrated with the same engine python environment to launch the Resource Mapping Tool. If it fails to launch the tool from the Editor, please follow above steps to make sure expected scripts and binaries are present.
```

Comment:
Note: the Editor is integrated with the same engine python environment *used* to launch the resource mapping tool. If the tool fails to launch from Editor...

Feel like this paragraph could use a rewrite as its slightly confusing to read, the intent is somewhat muddied IMHO